src/mfs.jl

```diff
@@ -0,0 +1,230 @@
+using BlackBoxOptim: bboptimize, best_candidate
+using Random: GLOBAL_RNG
+using StateSpaceSets: statespace_sampler
+export minimal_fatal_shock, MFSBruteForce, MFSBlackBoxOptim
+
+
+
+
+""" 
+    random_pertubation(mapper::AttractorMapper, X, search_area, dim,  total_iterations=10000)
+
+This function generates a random pertubation of the initial point "X" within specified "search_area" and checks if it is in the same basin of attraction.
+It does so by generating a random vector of length dim and then adding it to X.
+If the pertubation is not in the same basin of attraction, it calculates the norm of the pertubation and compares it to the best pertubation found so far.
+If the norm is smaller, it updates the best pertubation found so far.
+It repeats this process total_iterations times and returns the best pertubation found.
+`mapper`  one of available in Attractors.jl [`AttractorMapper`](@ref) constructed with respect to dynamical system
+`X` initial point to be tested
+`search_area` array of two points defining the search area
+`dim` dimension of the system
+`total_iterations` number of random pertubations to try, default = 10000
+
+"""
+
+
+function random_pertubation(mapper::AttractorMapper, X, search_area, dim, id_x;   total_iterations=10000)
```

Comment:
`random_pertubation` is an "incorrect" name. It doesn't state what the function does. The function doesn't generate a random perturbation. Probably a more appropriate name is `crude_initial_radius`